### PR TITLE
Add comment text area to technology register/edit summary

### LINKED ISSUE #465: Correções de bug - Formulários de Tecnologias - Aba Resumo
- Inserir um campo de comentário (textarea) ao final da página, antes dos aceites, para que o cadastrador possa enviar mensagem para o curador. Essa mensagem não precisa ficar registrada na tabela da revisão, visto que ela não será publicada. Será vista apenas pela curador no momento da avaliação.
- Adicionar esses checkbox para ser marcado no lugar da Política de Privacidade e Termos de Uso de Serviços:
   - Declaro ciência de que devo fornecer apenas informações verdadeiras no cadastro das tecnologias. Veja mais nos Termos e Condições de Uso. (OBRIGATÓRIO)
   - Estou ciente de que as informações cadastradas são de minha inteira responsabilidade, e a Plataforma Sabiá não responderá por quaisquer violações ao Direito de Propriedade Intelectual e Direito Autoral de terceiros. Veja mais nos Termos e Condições de Uso. (OBRIGATÓRIO)
   - Estou ciente de que poderei ser penalizado com advertência, suspensão e encerramento da minha conta por eventuais violações a direitos de terceiros no cadastro das tecnologias, como o Direito de Propriedade Intelectual e Direito Autoral. Veja mais nos Termos e Condições de Uso. (OBRIGATÓRIO)
   - Declaro ciência de que as transgressões a direitos de terceiros no cadastro das tecnologias podem implicar em responsabilização na esfera jurisdicional cível e criminal. Veja mais nos Termos e Condições de Uso. (OBRIGATÓRIO)

### PR DESCRIPTION
**Pull request by calaca**
## Summary

image

Resolves #465 

## Relevant technical choices

- Created a new service to fetch the current technology most recent comment

## QA Steps

1. Login
2. Create or edit an existing technology
3. At the last technology register step, verify that a section called "Comentários" exists and has a text area
4. Fill the text area with some comment you'd like to leave the reviewer
5. Save
6. Edit this technology again
7. Verify that the previously created comment is shown
8. Update the comment and save
9. Edit this technology again
10. Verify that the most recent comment is shown

## Checklist

- [x] My code is tested and passes existing unit tests.
- [x] My code has an appropriate set of unit tests which all pass.
- [x] My code passes the linting.
- [x] My code has proper inline documentation.
- [x] I have manually tested this PR.

### CODE REVIEW

File: packages/web/services/technology.js [hunk 1]
```diff
@@ -359,3 +359,25 @@ export const createTechnologyReview = async (data) => {
...
+
+	const response = await apiGet(`technologies/${id}/comments`);
+
+	if (response.status !== 200) {
+		return {};
+	}
+
+	const { data } = response;
+
+	return data[data.length - 1];
```
Comment (marcusmota) [inline packages/web/services/technology.js:382]: you can use data.pop(); here 👍

**Review by marcusmota — APPROVED** [review]
lgtm, left 1 small comment

File: packages/web/services/technology.js (hunk 1 above)
Comment (calaca) [reply packages/web/services/technology.js:382]: Yeah, but `pop` modifies the array. Is it something ok for us to do here?

Comment (luizeboli) [reply packages/web/services/technology.js:382]: My 2 cents: we should avoid mutability 😄

**Review by lcnogueira — APPROVED** [review]
Great job, Lorena 💯

**Pull request merged**